### PR TITLE
[RTM] Fix aCompCor

### LINKED ISSUE #177: PV of segmentation projected from T1 impedes aCompCor and CSF signal extraction
Improve segmentation. See screenshot; voxels labeled "CSF" overlap the cortex
screen shot 2016-11-08 at 4 41 05 pm

@chrisfilo

### PR DESCRIPTION
**Pull request by chrisfilo**
- use masking and erosion to avoid partial volumening effects
- don't provide mean singnal extracted only from the ventricles - this fails for subjects with very small ventricles
- add aCompCor ROI to reports

Closes #177

### CODE REVIEW

File: fmriprep/workflows/confounds.py [hunk 1]
```diff
@@ -56,13 +50,93 @@ def discover_wf(settings, name="ConfoundDiscoverer"):
...
+
+        combined = np.zeros_like(WM_data)
+
+        combined[WM_data != 0] = 1
+        combined[CSF_data != 0] = 1
+
+        new_nii = nb.Nifti1Image(combined, nb.load(ref_header).affine,
+                                 nb.load(ref_header).header)
+        new_nii.to_filename("logical_and.nii.gz")
+        return os.path.abspath("logical_and.nii.gz")
```
Comment (shoshber) [inline fmriprep/workflows/confounds.py]: Perhaps I am misreading the code but isn't this the logical or?

**Comment by chrisfilo** [comment]
This is waiting for a merge in niworkflows, but I'm pretty happy with the new masks:

image

image

**Comment by chrisfilo** [comment]
Still getting some SignalEstract header issues from a couple of subjects.

**Comment by chrisfilo** [comment]
I fixed all of the issues - this runs now on all Beast subjects.

**Comment by oesteban** [comment]
Maybe I should've merged this one first? Now there are some conflicts :(

**Comment by chrisfilo** [comment]
I've resolved the conflicts.

**Pull request merged**